docs/api-reference.md

```diff
@@ -229,6 +211,9 @@ Example:
 
 ## Deprecated API
 
+### connectionNormalizer.js and connectionMatcher.js
+These .js files are deprecated as of Tableau 2020.3. The <connection-normalizer> component can be added to the connectionResolver.tdr fie as shown in the connection-normalizer section above.
```

Comment:
Can you add the element/attribute names.  Technically the files could have been named anything before

Reply:
Added